mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/query/parsing/nativequery/NativeQueryParserTest.java

```diff
@@ -61,37 +56,25 @@ public void shouldParseComplexAggregateQuery() {
 		String unwind = "{'$unwind': '$clicks'}";
 		String group = "{ '$group': {'_id' : '$_id' ,'clicks' : {'$push':'$clicks'} ,'token' : { '$push': '$TOKEN' } } }";
 		String sort = "{ '$sort': { '_id' : -1 } }";
-		String nativeQuery = "db.UserFactualContent.aggregate(["
-						+ match
-						+ "," + unwind
-						+ "," + group
-						+ "," + sort
-						+ "])";
-		ParsingResult<MongoDBQueryDescriptorBuilder> run =  new ReportingParseRunner<MongoDBQueryDescriptorBuilder>( parser.Query() ).run( nativeQuery );
-
-		System.out.println( ParseTreeUtils.printNodeTree( run ) );
+		String nativeQuery = "db.UserFactualContent.aggregate([" + match + "," + unwind + "," + group + "," + sort + "])";
+		ParsingResult<MongoDBQueryDescriptor> run = new ReportingParseRunner<MongoDBQueryDescriptor>( parser.Query() ).run( nativeQuery );
 
-		MongoDBQueryDescriptor queryDescriptor = run.resultValue.build();
+		MongoDBQueryDescriptor queryDescriptor = run.resultValue;
 
 		assertThat( queryDescriptor.getCollectionName() ).isEqualTo( "UserFactualContent" );
 		assertThat( queryDescriptor.getOperation() ).isEqualTo( Operation.AGGREGATE_PIPELINE );
-		assertThat( queryDescriptor.getPipeline() )
-			.containsExactly(
-					Document.parse( match )
-					, Document.parse( unwind )
-					, Document.parse( group )
-					, Document.parse( sort )
-					);
+		assertThat( queryDescriptor.getPipeline() ).containsExactly( Document.parse( match ), Document.parse( unwind ), Document.parse( group ),
+				Document.parse( sort ) );
```

Comment:
Please avoid to reformat automatically our files. It makes it very difficult to find the real changes in there.

To fix it, use your IDE to compare both versions then bring back all the changes that are purely reformatting.

Thanks!

Reply:
Of course @gsmet ! I try to correct it

Reply:
fixed